.github/workflows/c.yaml

```diff
@@ -31,6 +31,8 @@ jobs:
 
     runs-on: ubuntu-latest
     container: ubuntu:18.04
+    env:
+      ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: true
```

Comment:
This was implemented in order to solve an error on the workflow. Discussion can be found [here](https://snav.slack.com/archives/C02E0A1CE3Z/p1720180486047209)